source/Willow/ComponentClassificationCommandBuilder.class.st

```diff
@@ -16,7 +16,14 @@ Class {
 { #category : #private }
 ComponentClassificationCommandBuilder >> commandStyledWith: aSymbol [
 
-	^ ComponentClassificationCommand toStyleWith: classificationNamespace >> aSymbol
+	^ self commandStyledWithAll: (Array with: aSymbol)
+]
+
+{ #category : #'as yet unclassified' }
+ComponentClassificationCommandBuilder >> commandStyledWithAll: aSymbolCollection [
+
+	^ ComponentClassificationCommand toStyleWithAll: (
+		aSymbolCollection collect: [:symbol | classificationNamespace >> symbol ])
 ]
```

Comment:
You can combine several styling command using `+` . So I see no need to allow for creating the command with several styles.

Reply:
Please rollback this changes. See the following comments.